### PR TITLE
The order of deletion option is wrong in htpasswd command.

### PR DESCRIPTION
**Pull request by Jooho**
`$ htpasswd </path/to/users.htpasswd> -D <user_name>` shows syntax error  so I just change the order of deletion option "-D" before htpasswd file.

### CODE REVIEW

File: admin_guide/configuring_authentication.adoc [hunk 1]
```diff
@@ -157,7 +157,7 @@ $ htpasswd </path/to/users.htpasswd> <user_name>
 To remove a login from the file, run:
 
 ----
-$ htpasswd </path/to/users.htpasswd> -D <user_name>
+$ htpasswd -D </path/to/users.htpasswd>  <user_name>
```
Comment (adellape) [inline admin_guide/configuring_authentication.adoc]: Not a huge deal, but there's an extra space before `<user_name>`.

**Comment by adellape** [comment]
One minor nit, otherwise LGTM. Thanks!

**Comment by Jooho** [comment]
Thanks!

**Comment by adellape** [comment]
:+1: Thanks.

**Pull request merged**